src/reducers/MapReducer.js

```diff
@@ -153,7 +162,8 @@ export default (state = INITIAL_STATE, action) => {
                 }, state.filterCriteriaKeys),
                 mapLoaded: existingMapLoadedState,
                 filterResultsScrutinyLoose: state.filterResultsScrutinyLoose,
-                selectedSite: state.selectedSite
+                selectedSite: state.selectedSite,
+                viewStyle: state.viewStyle
```

Comment:
GH-99 and GH-100 -- we weren't sending `viewStyle` when `INITIALIZE_MAP` came through, so we were always blowing away the `ListView` state in those cases.